zinc/src/test/scala/sbt/inc/LookupSpec.scala

```diff
@@ -93,17 +67,19 @@ class LookupSpec extends BaseIncCompilerSpec {
     lookup.lookupAnalysis("P1$P2") shouldEqual None
   }
 
-  it should "find correct analysis for class file" in mockedProjectContext { setup =>
-    import setup._
+  // it should "find correct analysis for class file" in mockedProjectContext { setup =>
+  //   import setup._
 
-    lookup.lookupAnalysis(project1.P1P2.classFile) shouldEqual project1.analysis
-    lookup.lookupAnalysis(project2.P1P2.classFile) shouldEqual project2.analysis
-    lookup.lookupAnalysis(project2.P2.classFile) shouldEqual project2.analysis
-    lookup.lookupAnalysis(project1.P1J2.classFile) shouldEqual project1.analysis
-    lookup.lookupAnalysis(project1.P1C2.classFile) shouldEqual project1.analysis
+  //   val lookup = new LookupImpl(config)
```

Comment:
https://github.com/sbt/zinc/blob/1.0/internal/zinc-core/src/main/scala/sbt/internal/inc/Lookup.scala#L33 It says that this is class file.

When class foo.A is used in code that comes from directoty (not jar) IncrementalCommon.externalBinaryModify calls lookupAnalysis with class file not directory. 

Tested in zinc-intellij integration.